zeekscript/formatter.py

```diff
@@ -1207,24 +1207,20 @@ def format(self):
             self._write_sp()
             self._format_child()  # <expr>
 
-        elif ct1 in ("{", "["):
-            # Vector/table/set initializers: '['/'{' <expr_list> ']'/'}'
-            do_linebreak = self.is_complex()  # Must call before we consume opener
-            self._format_child(hints=Hint.NO_LB_BEFORE)  # '{' / '['
-            if self._get_child_name() == "expr_list":
-                if do_linebreak:
-                    self._write_nl()
-                    self._format_child(hints=Hint.COMPLEX_BLOCK)  # expr_list
-                    self._write_nl()
-                else:
-                    self._write_sp()
-                    self._format_child()  # expr_list
-                    self._write_sp()
+        elif ct1 in ["{", "["]:
+            # Vector/table/set initializers: '{'/'[' <expr_list> '}'/']'
+            # Curly brace inits with multiple elements should linebreak
+            multi_ele_curly = len(self._get_child(1).children) > 1 and ct1 == "{"
+            do_linebreak = self.is_complex() or multi_ele_curly
+            self._format_child(hints=Hint.NO_LB_BEFORE)  # '{'/'['
+            if do_linebreak:
+                self._write_nl()
+                self._format_child(hints=Hint.COMPLEX_BLOCK)  # Inner expression(s)
+                self._write_nl()
             else:
-                # Just a space when the initializer list has no members.
-                self._write_sp()
+                self._format_child()  # Inner expression(s)
 
-            self._format_child()  # '}' / ']'
+            self._format_child()  # '}'/']'
```

Comment:
Petty! 🤡

Reply:
When I changed to apply from just `[` to `{` as well I just manually brought the comments back rather than going with `main` - another fun one is making the order of `[` and `{` consistent in comments like:

``` diff
- # Vector/table/set initializers: '['/'{' <expr_list> ']'/'}'
+ # Vector/table/set initializers: '{'/'[' <expr_list> '}'/']'
```

¯\\\_(ツ)_/¯